### PR TITLE
Added test docker compose

### PR DESCRIPTION
**Pull request by ViMuilu**
### Description

Added docker-compose test.

### Type of change

- [x] New feature (non-breaking change which adds functionality)

### Changes Made

docker-compose test disables https checks and changes aggregator/config/registries url from http://localhost:8080/services to http://app_registry:8080/services.

### CODE REVIEW

File: registry/registry.py [hunk 1]
```diff
@@ -45,25 +50,49 @@ async def service_types(request):
...
-    """POST request to the /services endpoint.
+if CONFIG.test is False:
 
-    Register a new service at host.
-    """
-    LOG.debug("POST /services received.")
-    # Tap into the database pool
-    db_pool = request.app["pool"]
+    @routes.post("/services")
+    @validate(load_schema("self_registration"))
```
Comment (blankdots) [inline registry/registry.py:56]: not ideal to have some special code just for test.
We can probably replace the `https://github.com/CSCfi/beacon-network/blob/master/registry/schemas/self_registration.json` directly in the container with something that accepts `http` as well

How can this be done:
- create a new self_registration.json where you modify the pattern to work with http
- place that file in `testconfig` folder
- in docker-compose under volumes for registry you can have a line `./testconfig/self_registration.json:/usr/local/lib/python3.6/site-packages/registry/schemas/self_registration.json`
- remove this `Config.test` from line 53 and the `test` from config
- the `config.ini` from test config can still be used to configure the registry and aggregator

**Review by blankdots — COMMENTED** [review]
It might be a easier approach to overwrite the configuration with a new one than to create custom code to deal with tests

File: registry/registry.py (hunk 1 above)
Comment (ViMuilu) [reply registry/registry.py:56]: These https checks have to be disabled when developing locally. So having the code like this speeds up development but if i have to always build the docker containers for these to take effect it really doesn't speed things up.

I should have explained this earlier but i forgot. Maybe i should change the env test to development.

Comment (blankdots) [reply registry/registry.py:56]: Then I would do something different in the schema we will allow all URLs and validate in the code if the URL fits the scheme HTTPS for prod and HTTP for development

You would need to check here: https://github.com/CSCfi/beacon-network/blob/dev/registry/endpoints/services.py#L19 I think

**Comment by ViMuilu** [comment]
I made some changes to the checks. I'm not sure if this is what you had in mind but hopefully they are a bit closer to that.

**Review by blankdots — APPROVED** [review]

**Comment by ViMuilu** [comment]
there might be a bug don't merge this yet

**Comment by ViMuilu** [comment]
now everything should work

**Pull request merged**